packages/test/test-service-load/src/loadTestDataStore.ts

```diff
@@ -36,11 +36,24 @@ export interface ILoadTest {
 	run(config: IRunConfig, reset: boolean): Promise<boolean>;
 	detached(config: Omit<IRunConfig, "runId" | "profileName">): Promise<LoadTestDataStoreModel>;
 	getRuntime(): Promise<IFluidDataStoreRuntime>;
+	waitLoaded(): Promise<void>;
+	/**
+	 * Generate ops.
+	 * @param n - number of ops to generate
+	 */
+	generateChanges(n: number);
```

Comment:
do we actually need to generate ops specifically for stashed ops. usually for this tests we try to keep features isolated, as it is more about leveraging features together, than stressing a specific feature in a specific way.